src/components/PersonLink.tsx

```diff
@@ -0,0 +1,17 @@
+import cn from 'classnames';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+interface Props {
+  person: Person;
+}
+
+export const PersonLink:FC<Props> = ({ person }) => (
+  <Link
+    to={`/people/${person.slug}`}
+    className={cn({ 'has-text-danger': person.sex === 'f' })}
```

Comment:
Before starting to work with functions and components, it would be very cool to define the variables that we use further into separate constants, so that we do not use magic values:
```
const FEMALE = 'f';
```
This will provide us with better readability of the code itself, conditions in the code and understanding that we will not accidentally make a mistake in a line.
This applies to all code.